app/lib/BaseEditorController.php

```diff
@@ -737,19 +751,24 @@ public function SummaryDisplay($pa_options=null, $pb_ajax_load_displays=FALSE) {
 	 * @param bool $pb_ajax_load_displays Load display using AJAX?
 	 * @return bool
 	 */
-	public function SummaryData($pa_options=null, $pb_ajax_load_displays=FALSE) {
+	public function SummaryData($pa_options=null) {
```

Comment:
Moved reused code into a separate function to make it easier to see which parts are specific to the 'section' we are collecting info for.